eval/src/vespa/eval/tensor/dense/dense_tensor_address_combiner.cpp

```diff
@@ -66,4 +71,24 @@ DenseTensorAddressCombiner::combineDimensions(const eval::ValueType &lhs,
             eval::ValueType::tensor_type(std::move(result)));
 }
 
+
+CommonDenseTensorCellsIterator::CommonDenseTensorCellsIterator(const Mapping & common,
+                                                               const Mapping & right,
+                                                               const eval::ValueType &type_in,
+                                                               CellsRef cells)
+    : _type(type_in),
+      _cells(cells),
+      _address(type_in.dimensions().size(), 0),
+      _common(common),
+      _mutable(right),
```

Comment:
I suggest renaming _mutable to _right to preserve semantics across classes

Reply:
Fixed